libs/block-editor/src/lib/extensions/components/suggestions-list-item/suggestions-list-item.component.scss

```diff
@@ -1,10 +1,15 @@
 :host {
+    box-sizing: border-box;
     display: flex;
     gap: 1rem;
     padding: 0.25rem 0.5rem;
     background-color: #fff;
     cursor: pointer;
     align-items: center;
+    height: 40px;
```

Comment:
Why do we need this?

Reply:
Suggestions-list-item had a non-explicit `height` of `39.Xpx`, I rounded it to 40px per list item to set a `max-height` of `400px` on the suggestion-list container to be able to display up to 10 options and more than that must be scroll.